### ISSUE TITLE
Support custom babel configuration

### ISSUE THREAD

**Issue by sHooKDT** [laget-se/react-gettext-parser#39]
A lot of people are using various proposal plugins for babel. react-gettext-parser is unable to parse files using some additional syntax.
We need to just add more plugins to `.babelrc` to support common cases or, more preferably add support for custom configuration.

**Comment by alexanderwallin** [comment]
Yes, this would make perfect sense. Ideally, `react-gettext-parser` would just pick up whatever Babel configuration is available.

**Comment by sHooKDT** [comment]
@alexanderwallin can we just add some proposal plugins for now, so I can start using this tool, and then start working on auto picking up project's .babelrc?
It fails on parsing "export default from" and maybe "export * from" for now.

**Comment by alexanderwallin** [comment]
If that's easy to setup, then sure. Would you be able to submit a PR?

**Comment by sHooKDT** [comment]
Yes, of course. I'll try when I get some time.

**Comment by sHooKDT** [comment]
Finally I found some time to make a PR. Now it is possible to parse my entire project.
I made a quick research about picking up external babelrc.
The thing is .babelrc and this plugins list are 2 different kind of things. Here we specify plugins used for parsing (for understand syntax), but in .babelrc files we see a list of transformations that will be applied, not a list of used syntactic constructions.
For example those users who support only latest browsers (chrome canary) can leave this list empty at all. And if we'll make a list of plugins according to .babelrc, out parser will not be able to parse code at all.
The same issue exists in babel-eslint project (https://github.com/babel/babel-eslint/issues/573).
Seems like it is not being solved for almost a year. But looks like they have added an option to specify extra plugins used alongside with predefined.
